TESTS WITH ERRORS 
-------------------------------
TESTS FAILED  
test10 (gradingtests.UnitTests)
Traceback (most recent call last):
  File "/Users/k2i/academics/automatedgradingapp/app/testing/kb-inference/gradingtests.py", line 282, in test10
    self.assertFalse(answer2)
AssertionError: True is not false

-------------------------------
TESTS SKIPPED 
-------------------------------
